app/scripts/components/common/card/index.tsx

```diff
@@ -287,16 +289,17 @@ function CardComponent(props: CardComponentPropsType) {
     const { linkProperties: linkPropertiesProps } = props;
     linkProperties = linkPropertiesProps;
   } else {
-    const { linkTo, onLinkClick,  } = props;
+    const { linkTo, onLinkClick, isLinkExternal } = props;
     linkProperties = {
       linkTo,
       onLinkClick,
       pathAttributeKeyName: 'to',
-      LinkElement: SmartLink
+      LinkElement: SmartLink,
+      isLinkExternal
     };
   }
 
-  const isExternalLink = /^https?:\/\//.test(linkProperties.linkTo);
+  const isExternalLink = linkProperties.isLinkExternal ?? /^https?:\/\//.test(linkProperties.linkTo);
 
   return (
     <ElementInteractive
```

Comment:
Will you need to pass in`isExternalLink` to linksProps of `ElementInteractive` ? that way `SmartLink will get that attribute passed to it from there as well

```    
<ElementInteractive
      linkProps={{
        as: linkProperties.LinkElement,
        [linkProperties.pathAttributeKeyName]: linkProperties.linkTo,
        onLinkClick: linkProperties.onLinkClick,
        isExternalLink: isExternalLink
      }}
      as={CardItem}
      cardType={cardType}
      className={className}
      linkLabel={linkLabel ?? 'View more'}
      onClickCapture={onCardClickCapture}
    >